src/amo/components/RatingManagerNotice/index.js

```diff
@@ -28,7 +28,7 @@ const RatingManagerNotice = ({
 
   if (type) {
     return (
-      <Notice type={type} {...props}>
+      <Notice type={type} light {...props}>
```

Comment:
I think it would be a nice feature if *Delete rating* gets hidden when the *Saved* message is visible. This would make the screen less jumpy.

So, instead of:

<img width="358" alt="screenshot 2018-09-25 14 18 40" src="https://user-images.githubusercontent.com/55398/46037924-59134b00-c0cf-11e8-8e71-a4dfa22c3147.png">


...make it look like:

<img width="358" alt="screenshot 2018-09-25 14 27 32" src="https://user-images.githubusercontent.com/55398/46037930-5d3f6880-c0cf-11e8-9e8e-3956a044ffc8.png">

I guess it would require a new property on `AddonReviewCard` like `hideControls` or something.

What do you think?

Reply:
Sounds like a good idea. I've implemented this. I had to monkey around with some padding to remove any jumpiness, and it looks fine when I run it locally now.